converse.js

```diff
@@ -2427,7 +2428,15 @@
                 this.toggleOccupants();
                 return this;
             },
-
+			
+            toggleCall: function (ev) {
+                ev.stopPropagation();
+                converse.emit('callButtonClicked', {
+                    connection: converse.connection,
+                    model: this.model
+                });
+            },
+			
```

Comment:
I don't think this function is necessary because `ChatRoomView` extends `ChatBoxView` and therefore inherits the `toggleCall` method from there.